jerry-debugger/src/lib/protocol-handler.ts

```diff
@@ -119,6 +123,40 @@ export class JerryDebugProtocolHandler {
     this.lastBreakpointExact;
   }
 
+  setDebugger(debuggerClient: JerryDebuggerClient) {
+    this.debugger = debuggerClient;
+  }
```

Comment:
Does it make sense to pass this into the constructor instead?
No need to re-assign it later I think?

Reply:
nit: Why a method and not just a public property?

Reply:
I grew up in C++ land when that was considered bad form. :) But sure...